Automattic/sensei
# Log when extensions are installed or updated on the Extensions page

### Changes proposed in this Pull Request
Logs when extensions are installed or updated on the Extensions page.

### Testing instructions

- Enable usage tracking.
- On the Extensions page, click the _Update_ button on a card.
- Ensure `sensei_extensions_update` is logged and `slug` is set appropriately.
- On the Extensions page, click the _Install_ button on a card.
- Ensure `sensei_extensions_install` is logged and `slug` is set appropriately.

TODO
- [ ] Update notifications
